CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # `ipyradiant` CHANGELOG
 
+## 0.1.2 (TODO) (2021-01-21)
```

Comment:
Need to update this and include additional changes from https://github.com/jupyrdf/ipyradiant/pull/84